src/css/flags/overlay.less

```diff
@@ -5,4 +5,9 @@
     .jw-controls-right .jw-logo {
         display: none;
     }
+    &-sharing, &-related {
```

Comment:
Cool, I will merge sharing and related PRs. Just need this one to get the package.json commit out.